misc/chrome_extension/content-script.js

```diff
@@ -172,33 +172,35 @@ window.onload = function () {
 	 */
 	function mouseoverHandler(event) {
 
-		if (!ableToMouseOver) {
+		if (!__ableToMouseOver) {
 			return;
 		}
 
 		if (event.target === document.body ||
-			(prev && prev === event.target)) {
+			(__previousSelecetedElement && __previousSelecetedElement === event.target)) {
 			return;
 		}
 
+		//Removes the __highlight class from the last element
 		try {
-			if (prev) {
-				prev.className = prev.className.replace(/\b__highlight\b/, '');
-				prev = undefined;
+			if (__previousSelecetedElement) {
+				__previousSelecetedElement.className = __previousSelecetedElement.className.replace(/\b__highlight\b/, '');
+				__previousSelecetedElement = null;
```

Comment:
You can move the `= null`, _after_ the catch (){} . and remove it from here and the catch()
it will make it obvious it's always reseted.